### PR TITLE
News Channel Support

### PR DESCRIPTION
**Pull request by FrostiiWeeb**
## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/guild.py [hunk 1]
```diff
@@ -1188,6 +1188,7 @@ async def create_text_channel(
         topic: str = MISSING,
         slowmode_delay: int = MISSING,
         nsfw: bool = MISSING,
+        news: bool = MISSING,
```
Comment (bijij) [inline discord/guild.py:1191]: This could just default to `False`
```suggestion
        news: bool = False,
```

**Review by bijij — CHANGES_REQUESTED** [review]

**Review by Rapptz — CHANGES_REQUESTED** [review]
This needs documentation.

File: discord/guild.py [hunk 2]
```diff
@@ -1255,6 +1256,8 @@ async def create_text_channel(
             The maximum value possible is `21600`.
         nsfw: :class:`bool`
             To mark the channel as NSFW or not.
+        news: :class:`bool`
+            Whether to mark the channel as a news channel or not.
```
Comment (Rapptz) [inline discord/guild.py:1260]:
```suggestion
        news: :class:`bool`
            Whether to create the text channel as a news channel.

            .. versionadded:: 2.0
```

**Review by Rapptz — CHANGES_REQUESTED** [review]
This is failing CI due to a type check error.

Chances are, you need an overload in `_create_channel` handle the union of the two literal types being passed in.

**Comment by Rapptz** [comment]
Closing per request.

**Pull request closed without merging**